cabot/metricsapp/api/grafana_elastic.py

```diff
@@ -209,7 +209,8 @@ def _add_aggs(search_aggs, series, min_time, default_interval):
         if bucket is None:
             raise ValidationError('Cannot find metric with id {} for pipeline metric'.format(pipeline_agg))
 
-        search_aggs.pipeline(metric_name, metric_type, buckets_path=bucket)
+        metric_settings = metric.get('settings', {})
```

Comment:
Is passing `metric['settings']` new behavior here? What can this dict contain?

Reply:
It's setting for moving average (like the way of calculating it, how many windows are used, etc). Before this moving average wouldn't have worked in Cabot the same way as Grafana if we tried to make moving average queries. I verified with test queries that the Cabot query now has the same output as the Grafana one